js/reducers/index.js

```diff
@@ -174,14 +197,23 @@ function report(state = INITIAL_REPORT_STATE, action) {
       return state.set("answers", Immutable.fromJS(preprocessAnswersJSON(action.response)));
     case SET_NOW_SHOWING:
       return state.set("nowShowing", action.value);
-    case SET_QUESTION_SELECTED:
-      return state.setIn(["questions", action.key, "selected"], action.value);
+
+    // The following actions only trigger API middleware hooks that call out
```

Comment:
I guess this comment is outdated now since these functions call something else than middleware?